packages/core/src/text/Text.tsx

```diff
@@ -37,6 +37,7 @@ export type TextProps<T extends ElementType> = PolymorphicComponentPropWithRef<
      * Change text color palette
      */
     variant?: "primary" | "secondary";
+    color?: "primary" | "secondary";
```

Comment:
We should copy Icon here

https://github.com/jpmorganchase/salt-ds/blob/5e48416c048e474b3750f405d677b608853b10ef/packages/icons/src/icon/Icon.tsx#L20

Reply:
Needful has been done